examples/hello_compute_c/main.c

```diff
@@ -0,0 +1,139 @@
+#include "./../../ffi/wgpu.h"
+#include <stdio.h>
+#include <stdlib.h>
+
+#define WGPU_TARGET_MACOS 1
+#define WGPU_TARGET_LINUX 2
+#define WGPU_TARGET_WINDOWS 3
+
+#define BLEND_STATES_LENGTH (1)
+#define ATTACHMENTS_LENGTH (1)
+#define RENDER_PASS_ATTACHMENTS_LENGTH (1)
+#define BIND_GROUP_LAYOUTS_LENGTH (1)
+
+WGPUByteArray read_file(const char *name) {
+    FILE *file = fopen(name, "rb");
+    fseek(file, 0, SEEK_END);
+    long length = ftell(file);
+    unsigned char *bytes = malloc(length);
+    fseek(file, 0, SEEK_SET);
+    fread(bytes, 1, length, file);
+    fclose(file);
+    return (WGPUByteArray){
+        .bytes = bytes,
+        .length = length,
+    };
+}
+
+void read_buffer_map(
+    WGPUBufferMapAsyncStatus status, const uint8_t *data, uint8_t *userdata) {
+    if (status == WGPUBufferMapAsyncStatus_Success) {
+        printf("Times: %s \n", data);
+    }
+}
+
+int main(int argc, char *argv[]) {
+
+    if (argc != 5) {
+        return 0;
+    }
+
+    int numbers[] = {
```

Comment:
- We want these numbers to be `uint32_t` to match Rust.
- We can declare `numbers` like `uint32_t numbers[]`
- We can use `strtoul(argv[1], NULL, 0)` to read each string to an unsigned integer for each argument (instead of `atoi`)